docs/lib/auth/overview.md

```diff
@@ -3,6 +3,18 @@ title: Concepts
 description: Learn more about the foundational auth concepts for cloud-based application and how they work with Amplify Framework.
 ---
 
+<amplify-code-block-switcher tab-heading-list="JavaScript, TypeScript">
```

Comment:
Can we infer the `tab-heading-list` from the order and type of the code block?

Reply:
Not without letting the first render happen, which means a flicker & prerender pains. Considering refactoring TOC as well so that we get a single clean paint.